download.py

```diff
@@ -3,10 +3,14 @@
 from subprocess import call
 import requests
 from requests.auth import HTTPBasicAuth
+import http.cookiejar
 
 
 def getIframeUrl(source, part):
-    r = requests.get(source)
+    cj = http.cookiejar.MozillaCookieJar('cookies.txt')
+    cj.load()
+
+    r = requests.get(source, cookies = cj)
```

Comment:
can you remove spaces around `=` ?
so `cookies=cj)` to be inline with pep8. currently this would violate error
```
E251 | unexpected spaces around keyword / parameter equals
```